src/util/fs.js

```diff
@@ -412,6 +412,18 @@ export async function walk(
   return files;
 }
 
+export async function fileSizeOnDisk(loc: string): Promise<number> {
+  const stat = await lstat(loc);
+
+  return _fileSizeOnDisk(stat);
+}
+
+export function _fileSizeOnDisk(fileStat: Object): number {
```

Comment:
Can we merge the two functions into one? I don't think we need both. And call it `getFileSizeOnDisk`?